ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -47,10 +47,14 @@ StatusSectionLayout {
             return "";
 
         const json = JSON.parse(community.tags);
-        const tagsArray = json.map(tag => {
-            return tag.name;
-        });
-        return JSON.stringify(tagsArray);
+        if (json) {
```

Comment:
Two suggestions here:
1. Either use a try/catch instead and return `""` in the catch, since `parse` can fail too if the json is not valid.
2. Otherwise, just to be cleaner, do `if (!json)` and early return `""`

Reply:
1. failed `parse` process should return `null`
2. I based on principle that first if should be the most useful

Reply:
If you try to `JSON.parse` somethig taht is not valid JSON, it trows an error.

As for principle, we don't really have it, but my personal preference is to have the least amount of code in brackets, so early returning is always my suggestion.

Reply:
okay, maybe it throws some error, but how to process it? and should it be processed?